apps/filebrowser/src/filebrowser/api.py

```diff
@@ -211,6 +211,25 @@ def rmtree(request):
   return HttpResponse(status=200)
 
 
+@error_handler
+def get_trash_path(request):
+  path = _normalize_path(request.GET.get('path'))
```

Comment:
Why are we getting a path in the request? What does it contain? Perhaps we can use a more descriptive name.

Reply:
Checking the [old logic](https://github.com/cloudera/hue/blob/2d06975913542e128f504b2d57324349b5c3b6fc/apps/filebrowser/src/filebrowser/views.py#L330-L344) and the HDFS REST API for `Get Trash Root`: https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/WebHDFS.html#Get_Trash_Root

It also send path but I am not sure how HDFS is using it to find the trash directory.

Reply:
ok, so this path param points to the resource being deleted. Still not obvious why that would be needed but at least I now know HDFS expects it. https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html#getTrashRoot-org.apache.hadoop.fs.Path-